kraken_frontend/src/views/workspace/workspace-finding/workspace-finding-table.tsx

```diff
@@ -70,7 +70,7 @@ export default function WorkspaceFindingTable({ onClickRow, onAuxClickRow, filte
             </div>
             <div
                 className="workspace-findings-table"
-                style={{ "--columns": "4em 6em 1fr 1fr 12em 0.5fr" } as Record<string, string>}
+                style={{ "--columns": "4em 6em 1fr 1fr 12em 0.5fr", "overflow-x": "auto" } as Record<string, string>}
```

Comment:
Would this not be better suited to be put into the class `workspace-findings-table` ?